pageserver/src/layered_repository.rs

```diff
@@ -2121,6 +2139,8 @@ impl LayeredTimeline {
 
         let mut layers = self.layers.write().unwrap();
         for l in image_layers {
+            self.physical_size
```

Comment:
I wonder if we should update the `PERSISTENT_BYTES_WRITTEN` metric here when creating an image layer similar to when we create a L0 delta layer